### PR TITLE
Completed javascript-conditionals exercise.

### PR DESCRIPTION
**Pull request by twoosam**
image

image

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+function categorizeAcidity(pH) {
+  if (pH < 0) {
+    return 'invlaid pH level';
+  } else if (pH < 7) {
+    return 'acid';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 14) {
+    return 'base';
```
Comment (shawnkost) [inline javascript-conditionals/main.js:64]: What about when `pH` is 14?

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    case 'wakko':
+      console.log("Wer'e the warner brothers!");
+      break;
+    case 'dot':
+      console.log("I'm cute!");
+      break;
+    default:
+      console.log('Goodnight Everyone!');
+  }
+}
```
Comment (shawnkost) [inline javascript-conditionals/main.js:82]: Instead of `console.log`ing, we should be `return`ing

File: javascript-conditionals/main.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      break;
+    case 'sci-fi':
+      console.log('Interstellar');
+      break;
+    default:
+      console.log(
+        'Genre not recognized. Choose between action, comedy, horror, drama, musical, or sci-fi.'
+      );
+  }
+}
```
Comment (shawnkost) [inline javascript-conditionals/main.js:109]: Instead of `console.log`ing, we should be `return`ing

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Tim! Just a few tweaks here

**Comment by twoosam** [comment]
image

image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**